site/specification/ORCv1.md

```diff
@@ -1297,7 +1297,7 @@ in a bloom filter is as follows:
   * position = combinedHash % m
 6. Set the position in bit set. The LSB 6 bits identifies the long index
    within bitset and bit position within the long uses little endian order.
-  * bitset[position >>> 6] \|= (1L << position);
+  * bitset[position >> 6] \|= (1L << (position % 64));
```

Comment:
Oh, sorry, @dirtysalt . I missed that the original was written in Java style.
- https://github.com/apache/orc/blob/master/java/core/src/java/org/apache/orc/util/BloomFilter.java#L275-L277
```java
public void set(int index) {
  data[index >>> 6] |= (1L << index);
}
```